backends/gstreamer/src/audio_sink.rs

```diff
@@ -43,19 +56,19 @@ impl GStreamerAudioSink {
 }
 
 impl GStreamerAudioSink {
-    fn set_audio_info(&self, sample_rate: f32, channels: u8) -> Result<(), ()> {
+    fn set_audio_info(&self, sample_rate: f32, channels: u8) -> Result<(), BackendError> {
         let audio_info = gst_audio::AudioInfo::new(
             gst_audio::AUDIO_FORMAT_F32,
             sample_rate as u32,
             channels.into(),
         ).build()
-            .ok_or(())?;
+            .ok_or(BackendError::AudioInfoFailed)?;
```

Comment:
This also can't possibly fail. The only way how this can fail if incompatible other settings (e.g. conflicting channel positions) are given

Reply:
This feels like it's possible to cause in a refactoring, so I'll leave this error in